src/views/v2/tickets/item/extra/available.php

```diff
@@ -24,9 +24,30 @@
 	return;
 }
 
+$available_count = $ticket->available();
+
+/**
+ * Allows hiding of "unlimited" to be toggled on/off conditionally.
+ *
+ * @since 4.11.1
+ *
+ * @var bool $show_unlimited    Whether to show the "unlimited" text.
+ * @var int  $available_count The quantity of Available tickets based on the Attendees number.
+ */
+$show_unlimited = apply_filters( 'tribe_tickets_block_show_unlimited_availability', true, $available_count );
+
+$context = [
+	'ticket'         => $ticket,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 10 space(s) between "'ticket'" and double arrow, but found 9.

<sub>hash: b1a3222555a28b7346b7c79d8e57d207</sub>